packages/editor-ui/src/views/TemplatesCollectionView.vue

```diff
@@ -21,18 +21,21 @@
 		<template v-if="!notFoundError" #content>
 			<div :class="$style.wrapper">
 				<div :class="$style.mainContent">
-					<div v-if="loading || (collection && collection.description)" :class="$style.markdown">
+					<div
+						v-if="loading || (collection && 'description' in collection)"
+						:class="$style.markdown"
+					>
 						<n8n-markdown
-							:content="collection && collection.description"
-							:images="collection && collection.image"
+							:content="collection && 'description' in collection && collection.description"
```

Comment:
nitpick - maybe`isFullTemplatesCollection` typeguard here might be clearer. Same for other places here where you use this pattern.